draft-ietf-radext-radiusdtls-bis.md

```diff
@@ -695,7 +695,7 @@ The reason is that the session is expected to be used for transport of RADIUS pa
 
 Any non-RADIUS traffic on that session means the other party is misbehaving and is a potentially security risk.
 Similarly, any RADIUS traffic failing authentication vector or Message-Authenticator validation means that two parties do not have a common shared secret.
-Since the shared secret is static, this again means the other party is misbehaving.
+Since the shared secret is static, this again means the other party is misbehaving or has been misconfigured.
```

Comment:
"misconfigured" would mean that it is acceptable to configure a shared secret. Since we mandate "radsec" resp. "radius/dtls" as shared secret, any deviation from that is (at least formally) not standard-compliant and thus misbehaving.

(I know that most RADIUS/TLS implementations still allow to configure a custom shared secret)

Reply:
I see your point, and I agree